src/assets/scss/components/_base.scss

```diff
@@ -1,203 +1,493 @@
+            }
+        }
+    }
+
+    .l-2 {
+        flex: 0 0 16.6667%;
+        max-width: 16.6667%;
     }
-};
 
-// tablet and low resolution
-@media (min-width: 740px) and (max-width: 1023px) {
+    .l-3 {
+        flex: 0 0 25%;
+        max-width: 25%;
+    }
+
+    .l-4 {
+        flex: 0 0 33.3333%;
+        max-width: 33.3333%;
+    }
+
+    .l-6 {
+        flex: 0 0 50%;
+        max-width: 50%;
+    }
+
+    .l-10 {
+        flex: 0 0 83.3333%;
+        max-width: 83.333%;
+    }
+
+    .l-12 {
+        flex: 0 0 100%;
+        max-width: 100%;
+    }
+}
+
+@include responsive(tablet) {
+    .m-2 {
+        flex: 0 0 16.6667%;
+        max-width: 16.6667%;
+    }
+
+    .m-3 {
+        flex: 0 0 25%;
+        max-width: 25%;
+    }
+
+    .m-4 {
+        flex: 0 0 33.3333%;
+        max-width: 33.3333%;
+    }
+
+    .m-6 {
+        flex: 0 0 50%;
+        max-width: 50%;
+    }
+
+    .m-10 {
+        flex: 0 0 83.3333%;
+        max-width: 83.333%;
+    }
+
+    .m-12 {
+        flex: 0 0 100%;
+        max-width: 100%;
+    }
+}
+
+@include responsive(tablet) {
     .wide {
         width: 644px;
-    };
-    .site-header{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-        .site-header_container{
+    }
+
+    ;
+
+    .site-header {
+        border-bottom: none !important;
+
+        .site-header_container {
             display: block;
-                .site-header-right{
-                    margin: 0 auto;
-                }
-        }
-        .header-top-right{
-            
+
+            .site-header-right {
+                margin: 0 auto;
+            }
         }
 
-        .main-navigation{
-            .main-nav_icon-list{
+        .header-top-right {}
+
+        .main-navigation {
+
+            .btn-menu {
+                display: block;
+            }
+
+            .main-nav_icon-list {
                 display: block;
                 font-size: 1.5em;
             }
-            
-            .main-nav-icon{
-                
-                >i{
+
+            .main-nav-icon {
+
+                >i {
                     font-size: 1.6em;
                 }
             }
-            
-            .menu-header-container{
-                .nav-menu{
+
+            .menu-header-container {
+                background: #ebebeb;
+                max-width: 350px;
+                padding-left: 20px;
+                margin-top: 20px;
+
+                .nav-menu {
                     flex-direction: column;
                     align-items: flex-start;
                     width: 350px;
-                    .header-search{
-                            display: none;
+
+                    .header-search {
+                        display: none;
                     }
-                    .menu-item{
-                        line-height: normal;
-                        padding: 20px;
+
+                    .menu-item {
+                        // padding: 20px;
```

Comment:
@vihan85 
Không nên để comment code.

Reply:
Hi @huulochuynh
This issue has been resolved

Thanks best & regards